datm/cime_config/buildnml

```diff
@@ -11,15 +11,21 @@
 
 import os, sys
 
-_CIMEROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..","..","..","..","..")
-sys.path.append(os.path.join(_CIMEROOT, "scripts", "Tools"))
+_CDEPS_CONFIG = os.path.join(os.path.dirname(os.path.abspath(__file__)), os.pardir,os.pardir,"cime_config")
+_CIMEROOT = os.environ.get("CIMEROOT")
+if _CIMEROOT is None:
+    raise SystemExit("ERROR: must set CIMEROOT environment variable")
+_LIBDIR = os.path.join(_CIMEROOT, "scripts", "Tools")
+sys.path.append(_LIBDIR)
+sys.path.append(os.path.join(_CDEPS_CONFIG))
```

Comment:
os.path.join takes at least 2 arguments.  Not needed here.

Reply:
I have changed these in all of the datamodels.